include/simdjson/portability.h

```diff
@@ -205,6 +216,8 @@ static inline void aligned_free(void *mem_block) {
   _aligned_free(mem_block);
 #elif defined(__MINGW32__) || defined(__MINGW64__)
   __mingw_aligned_free(mem_block);
+#elif defined(__MSYS__)
+  free(*(((void **)mem_block) - 1));
```

Comment:
You probably want to guard against mem_block being null.

Reply:
null check is at the top of the function

Reply:
since `free`, `_aligned_free`, and `__mingw_aligned_free` check for nullptr, I think it's better to remove nullptr check from `aligned_free` and  only check for when manual alignement

Reply:
> since free, _aligned_free, and __mingw_aligned_free check for nullptr, I think it's better to remove nullptr check from aligned_free and only check for it when doing manual alignment

This function has been deleted in another PR... but the reason why there is a check is that some sanitizers that people are using throw a warning when deleting/freeing  null pointers.

Reply:
> null check is at the top of the function:

Point taken.